### PR TITLE
Auto-launch browser for http mode

### PR DESCRIPTION
**Pull request by matthewjamesr**
Gents,

This PR implements auto-launching of user's default browser if http server starts successfully.

### CODE REVIEW

File: web/server.go [hunk 1]
```diff
@@ -141,6 +142,8 @@ func startServer(ctx context.Context, address string, router chi.Router) error {
 		return ctx.Err()
 	case <-t.C:
 		fmt.Printf("Web server running on %s\n", address)
+		fmt.Printf("Launching browser....\n")
```
Comment (ademuanthony) [inline web/server.go]: `fmt.Println("Launching browser....")`

File: web/server.go [hunk 2]
```diff
@@ -141,6 +142,8 @@ func startServer(ctx context.Context, address string, router chi.Router) error {
...
+		browser.OpenURL("http://" + address)
 		return nil
```
Comment (ademuanthony) [inline web/server.go:112]: `fmt.Println("Ready")`
`return nil`

**Review by ademuanthony — DISMISSED** [review]
few changes needed

File: web/server.go [hunk 3]
```diff
@@ -11,6 +11,7 @@ import (
 	"time"
 
 	"github.com/go-chi/chi"
+	"github.com/pkg/browser"
```
Comment (itswisdomagain) [inline web/server.go]: We aim to use external libraries only if the solution provided cannot be quite easily accomplished with standard go libraries. For this use case, we don't need an external library. See this [SO answer](https://stackoverflow.com/a/39324149/6086027) and this [GH gist](https://gist.github.com/hyg/9c4afcd91fe24316cbf0).

**Review by itswisdomagain — CHANGES_REQUESTED** [review]

**Review by itswisdomagain — APPROVED** [review]

**Review by ademuanthony — APPROVED** [review]

**Pull request merged**